nanoFramework.System.Net.Http/Http/System.Uri.cs

```diff
@@ -288,6 +329,27 @@ public string AbsoluteUri
             }
         }
 
+        /// <summary>
+        /// Gets the escaped URI fragment, including the leading '#' character if not empty.
+        /// </summary>
+        ///<value>A <itemref>String</itemref> containing the fragment if present.</value>
+        public string Fragment => _fragment ?? string.Empty;
```

Comment:
same here, why not initialize all those private properties to string.Empty so that you don't need that check. Looking at the code, I don't think they'll have another occasion to be null. I maybe missing something here.

Reply:
Same answer really, following an existing pattern unless there was reason to change it.  I should add that to my list of things to recheck when i'm going back over things.